client/src/components/feedback/send.js

```diff
@@ -0,0 +1,260 @@
+import React, { Component } from 'react';
+import { Button, Form } from 'react-bootstrap';
+import PropTypes from 'prop-types';
+import Api from '../../services/api';
+import '../../styles/feedback-send.css';
+import MyBadge from '../shared/my-batch';
+import CancelButton from '../shared/button/cancel';
+
+class FeedbackSend extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      description: '',
+      isLoading: false,
+      timerCount: 3,
+      showTimer: true,
+      tooShortFeedback: false,
+      tooLongFeedback: false,
+      feedbackSent: false,
+      feedbackNotSent: false,
+      inputDisabled: false,
+      attemptCount: false,
+      validInput: true,
+    };
+    this.onSubmit = this.onSubmit.bind(this);
+    this.handleChange = this.handleChange.bind(this);
+    this.tooShortFeedback = this.tooShortFeedback.bind(this);
+    this.tooLongFeedback = this.tooLongFeedback.bind(this);
+    this.sendApi = this.sendApi.bind(this);
+    this.cancel = this.cancel.bind(this);
+    this.showFeedbackSent = this.showFeedbackSent.bind(this);
+    this.showFeedbackNotSent = this.showFeedbackNotSent.bind(this);
+    this.showTimer = this.showTimer.bind(this);
+  }
+
+  onSubmit() {
+    if (this.state.validInput) {
+      if (this.state.description.length < 10) {
+        this.tooShortFeedback();
+      } else if (this.state.description.length > 400) {
+        this.tooLongFeedback();
+      } else {
+        this.sendApi();
+      }
+    }
+  }
+
+  setInputFalse() {
+    this.setState(() => {
+      this.setState({
+        validInput: false,
+      });
+    });
+  }
+
+  validateDescription() {
+    const { description } = this.state;
+    const regex = /(^[a-zA-Z1-9]+[a-zA-Z1-9. ]+)$/;
```

Comment:
When I insert a coma into a message, a warning message appears and I cannot press the button "Send". In the warning message stay, that i can use coma.

Reply:
It is very strange, as in my last commit 'comma' was used. 

Fixed it again. Thanks

Reply:
It works now, but why other punctuation marks are not allowed?

Reply:
Our 'zakazchik' doesn't tell us what symbols he wants to be allowed here, so I decided to keep it cheap and strong :)